tests/test_load_data.py

```diff
@@ -1,14 +1,24 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+test_load_data
+----------------------------------
+
+Tests for `data.load_data` module.
+"""
+
+
+import unittest
 from copy import deepcopy
 from inicheck.tools import cast_all_variables
 from inicheck.utilities import pcfg
-import unittest
-
-from smrf.framework.model_framework import can_i_run_smrf
 
+from smrf.framework.model_framework import can_i_run_smrf, run_smrf
 from tests.test_configurations import SMRFTestCase
 
-
-class TestLoadData(SMRFTestCase):
+@unittest.skip(" Skipping over MYSQL tests since the do not work outside of NWRC")
```

Comment:
The default then is to not run the test? Another way might be to override `setup` to check if the server exists. Then we'll get the benefit of the tests when we're on the NWRC network.

Reply:
I will add an if statement to the test. If were on the network we will run this else we will just pass the test successfully.

Reply:
I was able to get the mysql skip test to work without a connection to the network. This way it will skip and instead of passing when not connected. Otherwise locally it will run and pass.